### PR TITLE
remove year from highlights title

### PR DESCRIPTION
**Pull request by tanyaparker**
**Before**
screen shot 2018-11-30 at 1 27 10 pm

**After**
screen shot 2018-11-30 at 1 27 02 pm

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/views/home_highlights.haml [hunk 1]
```diff
@@ -3,11 +3,9 @@
     .col-md-8
       %br/
       %h2
-        Hour of Code
-        = campaign_date('year').to_i - 1
+        Hour of Code Highlights
       %a.twitter-grid{href: "https://twitter.com/codeorg/timelines/925449654308704256?limit=4?ref_src=twsrc%5Etfw?limit=4"}
-        The Hour of Code
-        = campaign_date('year').to_i - 1
+        Hour of Code Highlights
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/views/home_highlights.haml:8]: This Twitter feed only appears in English so need to localize.

I'm not sure where this second "The Hour of Code 2017" string appears. I don't see it on the page, but it does appear in the page source. Perhaps it's what shows up when the twitter feed doesn't load properly? So I decided to update it as well.

File: pegasus/sites.v3/hourofcode.com/views/home_highlights.haml [hunk 2]
```diff
@@ -3,11 +3,9 @@
     .col-md-8
       %br/
       %h2
-        Hour of Code
-        = campaign_date('year').to_i - 1
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/views/home_highlights.haml:7]: Ah, so we were showing previous year on purpose?

File: pegasus/sites.v3/hourofcode.com/views/home_highlights.haml (hunk 1 above)
Comment (breville) [reply pegasus/sites.v3/hourofcode.com/views/home_highlights.haml:8]: I believe you're right.  The page source has an `<a class="twitter-grid" ...` which gets replaced by an `<iframe ...` when this grid is loaded by JS.

File: pegasus/sites.v3/hourofcode.com/views/home_highlights.haml (hunk 2 above)
Comment (tanyaparker) [reply pegasus/sites.v3/hourofcode.com/views/home_highlights.haml:7]: Yep. We were originally going to change it to 2018 when we flipped to `actual-hoc`. Then we would need to make sure to keep it 2018 for `post-hoc` in December, but change it back to current_year - 1 in January. Which all seemed too complex for what this achieves, so simplifying by removing the year.

**Pull request merged**